src/Http/ApiClient.php

```diff
@@ -13,13 +13,27 @@
 use GuzzleHttp\Client;
 use GuzzleHttp\HandlerStack;
 use GuzzleHttp\Psr7\Uri;
+use phpDocumentor\Reflection\Types\Self_;
 use Psr\Http\Message\ResponseInterface;
 
 /**
  */
 class ApiClient
 {
     /**
+     * These are the endpoints where there is no need to have an active session for the request
```

Comment:
Please go for `Endpoints not requiring active session for the request to succeed` (less letters :P)